frontend/server/src/main/java/com/amazonaws/ml/mms/http/InferenceRequestHandler.java

```diff
@@ -139,7 +98,7 @@ private void handleInvocations(
             ChannelHandlerContext ctx, FullHttpRequest req, QueryStringDecoder decoder)
             throws ModelNotFoundException {
         String modelName = NettyUtils.getParameter(decoder, "model_name", null);
-        if ((modelName == null || modelName.isEmpty())) {
+        if ((modelName == null) || modelName.isEmpty()) {
```

Comment:
This parenthesis is not necessary

Reply:
I can remove it, but I added it because there was one unnecessary brace.